src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrDate = date.split(fromFormat[3]);
```

Comment:
The use of `fromFormat[3]` to split the date string assumes that the separator is always at index 3. This might not be the case if the format changes. Consider dynamically determining the separator or passing it as a separate parameter.